### PR TITLE
17950 - Display a credit being applied in transactions list

### PR DESCRIPTION
**Pull request by rodrigo-barraza**
*Issue #:*
https://github.com/bcgov/entity/issues/17950

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the sbc-auth license (Apache 2.0).

### CODE REVIEW

File: auth-web/src/composables/transactions-factory.ts [hunk 1]
```diff
@@ -73,6 +73,24 @@ export const useTransactions = () => {
       if (response?.data) {
         transactions.results = response.data.items || []
         transactions.totalResults = response.data.total
+        const transactionClone = [...transactions.results]
+        transactionClone.forEach((transaction: Transaction, i: number) => {
+          if (transaction.refundDate) {
```
Comment (ochiu) [inline auth-web/src/composables/transactions-factory.ts:78]: Is this too broad? there could be multiple states that the refundDate would be set.

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-account-dev--pr-3128-06pniayw.web.app

File: auth-web/src/resources/display-mappers/payment-type-display.ts [hunk 2]
```diff
@@ -12,5 +12,6 @@ export const paymentTypeDisplay = {
   [PaymentTypes.NO_FEE]: 'No Fee',
   [PaymentTypes.ONLINE_BANKING]: 'Online Banking',
   [PaymentTypes.PAD]: 'Pre-Authorized Debit',
-  [PaymentTypes.WIRE]: 'Wire Transfer'
```
Comment (seeker25) [inline auth-web/src/resources/display-mappers/payment-type-display.ts:15]: Should probably gun WIRE

File: auth-web/src/resources/display-mappers/payment-type-display.ts [hunk 3]
```diff
@@ -12,5 +12,6 @@ export const paymentTypeDisplay = {
...
+  [PaymentTypes.WIRE]: 'Wire Transfer',
```
Comment (seeker25) [inline auth-web/src/resources/display-mappers/payment-type-display.ts:15]: I don't think WIRE ever shows up, we use EFT for WIRE

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-account-dev--pr-3128-06pniayw.web.app

**Review by ochiu — APPROVED** [review]

**Pull request merged**